.travis.yml

```diff
@@ -172,6 +171,8 @@ before_install:
       echo "library_dirs = $TRAVIS_BUILD_DIR/openblas/usr/local/lib" >> site.cfg
       echo "include_dirs = $TRAVIS_BUILD_DIR/openblas/usr/local/include" >> site.cfg
       echo "runtime_library_dirs = $TRAVIS_BUILD_DIR/openblas/usr/local/lib" >> site.cfg
+      # remove a spurious gcc/gfortran toolchain install
+      rm -rf /usr/local/Cellar/gcc/9.2.0_2
```

Comment:
Something more robust to future updates, perhaps with a wildcard (?) might be better here. I'm open to suggestions or fancy `brew` commands. It was getting late so I went with something surgical initially.

The original failure is visible from the unpinning of multibuild here:

https://travis-ci.org/tylerjereddy/scipy/builds/641570340?utm_source=github_status&utm_medium=notification

Reply:
This is the basis of the removal:

```
dyld: Library not loaded: /usr/local/opt/isl/lib/libisl.21.dylib

  Referenced from: /usr/local/Cellar/gcc/9.2.0_2/libexec/gcc/x86_64-apple-darwin17/9.2.0/f951

  Reason: image not found
```

Reply:
Note that `CC=gcc-6` and so on is already set prior to that.